templates/model/spec.go

```diff
@@ -205,34 +206,38 @@ func (s *Step) UnmarshalYAML(n *yaml.Node) error {
 	// on the value of "action".
 	var unmarshalInto any
 	switch s.Action.Val {
-	case "print":
-		s.Print = new(Print)
-		unmarshalInto = s.Print
-		s.Print.Pos = s.Pos // Set an approximate position in case yaml unmarshaling fails later
+	case "append":
```

Comment:
Note for reviewers: I alphabetized this list because it was feeling disorganized.